<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1421</width>
    <height>927</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string extracomment="&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wenn dieser Button geclickt wird, Ã¤ndert sich der Suchalgorithmus, mit dem die Visualisierung betrieben wird.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;"/>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>2000</width>
     <height>2000</height>
    </size>
   </property>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderSpeed">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>230</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelSpeed">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelete">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete the Map</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelColorStart">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelColorEnd">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTextStart">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>start point</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTextEnd">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>end point</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelErklaerungWalls">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;draw walls: &lt;/span&gt;press any key on your mouse&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChangeAlgo">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Change algorythm</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlgo">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZoomOut">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZoomIn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QLabel" name="disclaimerLabel">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Disclaimer: &lt;/span&gt;Zooming in or zooming&lt;/p&gt;&lt;p&gt;out is just allowed in Mode: Start!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreateLabyrinth">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Create a 
complex 
Labyrinth</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelState">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Mode:&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt; Start&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
